.travis.yml

```diff
@@ -16,14 +17,11 @@ addons:
 
 env:
   global:
-  - VERSION="10.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
-  - TRANSIFEX_USER='transbot@odoo-community.org'
+  - VERSION="10.0" TESTS="0" LINT_CHECK="0"
   - WKHTMLTOPDF_VERSION="0.12.4"
-  - secure: ArnbVaF5+ry6zVysZ7HA9xcnQrodo8FsXXcon9yINTYRfDC9QEr+/mTLAPRl+lVLdtWV2GuGuX0vPYzBxFWpY3LR0BpKqXzx0G51s94zR2WWEmizYFzFhpAuIxoU4CYNckHFSUaPAQJhwB/pYx9+H/W6bMjG/VnZBq+AmBJ2Kh0=
-
-  matrix:
+   matrix:
   - LINT_CHECK="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
+  - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
```

Comment:
@renatonlima if we add this MAKEPOT="1", I believe .pot files will be generated and possibly .po files might be overriten by the weblate translation service. I believe that starting from v12 we will be quite compatible to use the Brazilian localization along with other localizations, so activating the weblate translation makes sense. I just fear it may override the existing .po files of the project. Really not sure about that. What do you think?

Reply:
@rvalyi,

In l10n-brazil we always create fields and objects names in english, but all labels are in portuguese, now in version 12.0 we are putting all labels also in English, For example in module l10n_br_base I generated a POT file with the module terms and also uploaded one file pt_BR.po already with the portuguese translations, weblate synchronized the translations and didn't overwrite it so I would say there is no problem.

Reply:
indeed we should have no problem https://twitter.com/SBidoul/status/1163876494105612288

Reply:
👍